training/distributed_training/pytorch/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,492 @@
+    "- Make sure you set the [appropriate inbound/output rules in the `security group`](https://docs.aws.amazon.com/fsx/latest/LustreGuide/limit-access-security-groups.html). Specifically, opening up these ports is necessary for SageMaker to access the FSx filesystem in the training job.\n",
+    "- Make sure `SageMaker IAM Role` used to launch this SageMaker training job has access to `AmazonFSx`.\n",
+    "\n",
+    "### Specify the FSx information for the training job\n",
+    "Go to your [FSx console](console.aws.amazon.com/fsx/) and access your recently created filesystem. In the **Summary** section, note the _File system ID_ and the _Mount name_. Also, in the **Network & security** tab, click on the Network Tnterface to display the _Subnet ID_ and _Security groups_. Use this information to complete the cell below and run it to configure the FSx input for your SageMaker training job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "00bc6fe9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure FSx Input for your SageMaker Training job\n",
+    "from sagemaker.inputs import FileSystemInput\n",
+    "\n",
+    "# FSx file system ID with your training dataset. Example: 'fs-0bYYYYYY'\n",
+    "file_system_id = \"<fsx_id>\"\n",
+    "# FSx path with your training data # Example: '/fsx_mount_name/imagenet'\n",
+    "file_system_directory_path = \"/<fsx_mount_name>/<path_to_data>\"\n",
+    "file_system_access_mode = \"rw\"\n",
+    "file_system_type = \"FSxLustre\"\n",
+    "train_fs = FileSystemInput(\n",
+    "    file_system_id=file_system_id,\n",
+    "    file_system_type=file_system_type,\n",
+    "    directory_path=file_system_directory_path,\n",
+    "    file_system_access_mode=file_system_access_mode,\n",
+    ")\n",
+    "# Specify the training data channel using the FSx filesystem. This will be provided to the SageMaker training job later\n",
+    "data_channels = {\"train\": train_fs}\n",
+    "\n",
+    "# The following variables will be used later in the notebook\n",
+    "fsx_subnets = [\"<subnet_id>\"]  # Should be the subnet used for FSx. Example: subnet-0f9XXXX\n",
+    "fsx_security_group_ids = [\n",
+    "    \"<security_group_id>\"\n",
+    "]  # Should be the security group used for FSx. sg-03ZZZZZZ"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "085c55c1",
+   "metadata": {},
+   "source": [
+    "## Prepare SageMaker Training Images\n",
+    "\n",
+    "SageMaker by default uses the latest [Amazon Deep Learning Container Images (DLC)](https://github.com/aws/deep-learning-containers/blob/master/available_images.md) PyTorch training image. In this step, we use it as a base image and install additional dependencies required for training EfficientNet model. Run the cells below to assign a name for your image and observe the dockerfile."
```

Comment:
should we talk about choice of base DLC here?

Reply:
I don't totally follow your suggestion. Are you saying we should give more details about the choices of DLC images available?